packages/esphome/flexispot_ek5.yaml

```diff
@@ -119,7 +119,7 @@ switch:
     name: "M"
     id: switch_m
     icon: mdi:alpha-m-circle
-    data: [0x9b, 0x06, 0x02, 0x20, 0x00, 0xac, 0xb8, 0x9d]
+    data: [0x9b, 0x06, 0x02, 0x00, 0xac, 0xac, 0xb8, 0x9d]
```

Comment:
I would need to double check this in my example. The M function works via ESPHome, thus not sure if my control box has slightly different commands.

Reply:
I have followed this table:
https://github.com/iMicknl/LoctekMotion_IoT#command-list

How should the M-Key actually work? Do I first click on a preset, then set the height and then click on the M-Key?

Reply:
It should work the same as with the physical controller. You set a height, than press the "M" button and than choose the preset to overwrite. 

I will check if my control panel works with the other M button, or if this wasn't documented properly for my control panel.

Reply:
If I look [here](https://github.com/staxDB/standing-desk-interceptor#button-presses), the original signal seems to have been correct. The main README needs to be updated here: https://github.com/iMicknl/LoctekMotion_IoT#command-list

Reply:
I have just tested it: Your original signal was correct, now the M-Key also works for me.